### DISCUSSION TITLE
Feature: Mute hashtags

### DISCUSSION THREAD

**Discussion by Ciantic** [mastodon/mastodon#21762] [General]
This is kind of simple and tangential to my another feature request: #21323 making live tooting to same hash tag easier.

I noticed this is already possible, with filters, but it probably could be integrated to hashtag view for easier usage.

Mute button to make it easier to use:

<img src="https://user-images.githubusercontent.com/64731/204167979-0665d269-c7d9-41f7-8653-aa830fd539e9.png" width="500" />

It should allow muting and unmuting a tag.

It would got add a new filter:
<img src="https://user-images.githubusercontent.com/64731/204167939-c092f61f-be70-465f-b3e7-0928c02c4c02.png" width="500" />

Or remove the filter.

**Comment by Wurzelmann** [comment]
+1 would love this as a feature

**Comment by ThisIsMissEm** [comment]
As there's currently a bug with "hashtag" keyword filters, where it incorrectly ends up filtering the hash fragments of URLs as well, there may be interest in implementing this: https://github.com/mastodon/mastodon/pull/27991
